geometry_msgs/WrenchStamped data
---
bool success
